compiler/circle-mpqsolver/src/CircleMPQSolver.cpp

```diff
@@ -74,6 +80,67 @@ int entry(int argc, char **argv)
     setenv("LUCI_LOG", "100", 0);
   }
 
-  // TODO Run Bisect method
+  auto data_path = arser.get<std::string>("--data");
+  auto input_model_path = arser.get<std::string>("--input_model");
+  auto output_model_path = arser.get<std::string>("--output_model");
+
+  float qerror_ratio = arser.get<float>("--qerror_ratio");
+  if (qerror_ratio < 0.0 || qerror_ratio > 1.f)
+  {
+    std::cerr << "ERROR: quantization ratio should be above 0 and below 1" << std::endl;
```

Comment:
The if condition allows 0 and 1. So it would be better to update the error message.

```suggestion
    std::cerr << "ERROR: quantization ratio must be in [0, 1]" << std::endl;
```

Reply:
ok